ocaml/idl/datamodel.ml

```diff
@@ -7389,6 +7396,8 @@ let vm_guest_metrics =
       field ~qualifier:DynamicRO ~ty:DateTime "last_updated" "Time at which this information was last updated";
       field ~in_product_since:rel_orlando ~default_value:(Some (VMap [])) ~ty:(Map(String, String)) "other_config" "additional configuration";
       field ~qualifier:DynamicRO ~in_product_since:rel_orlando ~default_value:(Some (VBool false)) ~ty:Bool "live" "True if the guest is sending heartbeat messages via the guest agent";
+      field ~qualifier:DynamicRO ~lifecycle:[Published, rel_dundee, "New, to be used where relevant instead of checking PV driver version."] ~ty:tristate_type ~default_value:(Some (VEnum "unspecified")) "can_use_hotplug_vbd" "The guest's statement of whether it will start using a hotplugged VBD.";
```

Comment:
It's in the lifecycle, not in the description.
The description is `""The guest's statement of whether it will start using a hotplugged VBD."`